frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "axios": "^0.19.0",
     "eslint": "^6.6.0",
     "eslint-plugin-react": "^7.17.0",
+    "js-file-download": "^0.4.12",
```

Comment:
Ha! I worked with the guy that created this lib when he created it. Glad to see it being it used.